server/common/utils/uns.py

```diff
@@ -6,6 +6,8 @@
 
 from server.common.constants import SPATIAL_IMAGE_DEFAULT_RES
 
+import pyvips
```

Comment:
use `pyvips` to create DZI file and tiles

NOTE that you'll need to `brew install vips` to make sure `pyvips` works!